sdk/.pre-commit-config.yaml

```diff
@@ -30,14 +30,14 @@ repos:
       name: buildifier
       language: system
       require_serial: true
-      entry: "bazel run //:buildifier-pre-commit -- -mode=fix -v=true"
+      entry: "bash -c 'cd sdk/; bazel run //:buildifier-pre-commit -- -mode=fix -v=true'"
       types: [bazel]
     - id: pprettier
       name: pprettier
       language: system
       require_serial: true
       # NB: we need to pass a single argument which ends up in $0 when invoking bash -c
-      entry: "bash -c 'yarn install --silent && yarn run pprettier --write \"$@\"' bash"
+      entry: "bash -c 'yarn install --silent && files=(\"$@\") && yarn run --cwd=sdk pprettier --write ${files[@]##sdk/}' bash"
```

Comment:
* `files=(\"$@\")` creates a bash array out of the arguments (the discovered `.ts` and `.tsx` files).
* ` --cwd=sdk` makes yarn run in the `sdk/` dir, which is where the `package.json` file lives.
* `${files[@]##sdk/}` is an operation which removes the `sdk/` prefix from each element of the array.